recipes/property-manager/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "property-manager" %}
+{% set version = "2.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "19e1f230f95691ae74123fa43c6928d0f0cc464e841d6375941cda4a0585162f"
+
+build:
+  number: 0
```

Could it be ``noarch``?
```suggestion
  number: 0
  noarch: python
```
https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python